src/components/Pad/Pad.js

```diff
@@ -29,9 +29,12 @@ const PadStyled = styled.div`
   position: relative;
   border: 2px solid;
   border-color: ${props =>
-    props.isTriggered ? 'var(--color-pad-triggered)' : 'var(--color-dark)'};
+    props.isTriggered
```

Comment:
It seems to have a problem with the trigger. The style doesn't trigger properly, looks like it does randomly, sometimes it works, sometimes not.